### DISCUSSION TITLE
现在都是使用大陆白名单吗？

### DISCUSSION THREAD

**Discussion by wyfang** [shadowsocks/shadowsocks-windows#3177] [Q&A]
我不知道这样是否最好，但是就目前来看，经常steam平台更新游戏，发现走这边的流量，等发现的时候已经好几个小时了，而原本应该只要10分钟就能下载好的东西。

**Comment by wyfang** [comment]
事态越发不可控，白名单，导致几乎电脑上的软件，都会走SS了，因为软件并不流行，都是一些小软件，就变得访问很困难。

**Comment by chenshaoju** [comment]
你好，目前使用的是GeoSite，由V2Fly提供。

如果你希望你使用的软件经过代理，可以访问 https://github.com/v2fly/domain-list-community 提交你所使用的软件的域名，使其添加到列表中。

参考：https://github.com/v2fly/domain-list-community/pull/418

**Reply by wyfang** [reply]
找到了在gui-config.json中修改`"geositePreferDirect": true,`
但是为什么，当我改成true后，例如steam，下载游戏的时候，下载游戏依然会通关软件流量？日志中显示
```
2022-01-06 18:57:01.2348|DEBUG|Shadowsocks.Controller.TCPHandler|connect to 117.67.227.34:8088
2022-01-06 18:57:01.2348|DEBUG|Shadowsocks.Controller.TCPHandler|Socket connected to ss server: :abc.com:800
```
